com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1559,15 +1559,32 @@ ref _cullingResults
 
-                        hdCamera.parentCamera = parentCamera; // Used to inherit the properties of the view
+                        hdCamera.SetParentCamera(hdParentCamera); // Used to inherit the properties of the view
+                        hdCamera.resetPostProcessingHistory = false;
 
-                        if (visibleProbe.type == ProbeSettings.ProbeType.PlanarProbe && hdCamera.frameSettings.IsEnabled(FrameSettingsField.ExposureControl))
+                        if (visibleProbe.type == ProbeSettings.ProbeType.PlanarProbe)
                         {
-                            RTHandle exposureTexture = GetExposureTexture(hdCamera);
-                            visibleProbe.RequestProbeExposureValue(exposureTexture);
-                            // If the planar is under exposure control, all the pixels will be de-exposed, for the other skies it is handeled in a shader.
-                            // For the clear color, we need to do it manually here.
-                            additionalCameraData.backgroundColorHDR = additionalCameraData.backgroundColorHDR * visibleProbe.ProbeExposureValue();
+                            //cache the resolved settings. Otherwise if we use the internal probe settings, it will be the wrong resolved result.
+                            visibleProbe.ExposureControlEnabled = hdCamera.frameSettings.IsEnabled(FrameSettingsField.ExposureControl);
+                            if (visibleProbe.ExposureControlEnabled)
+                            {
+                                RTHandle exposureTexture = GetExposureTexture(hdParentCamera);
+                                hdParentCamera.RequestGpuExposureValue(exposureTexture);
+                                visibleProbe.SetProbeExposureValue(hdParentCamera.GpuExposureValue());
+                                additionalCameraData.deExposureMultiplier = 1.0f;
+
+                                // If the planar is under exposure control, all the pixels will be de-exposed, for the other skies it is handeled in a shader.
+                                // For the clear color, we need to do it manually here.
+                                additionalCameraData.backgroundColorHDR = additionalCameraData.backgroundColorHDR * visibleProbe.ProbeExposureValue();
+                            }
+                            else
+                            {
+                                //the de-exposure multiplier must be used for anything rendering flatly, for example UI or Unlit.
+                                //this will cause them to blow up, but will match the standard nomralized exposure.
+                                hdParentCamera.RequestGpuDeExposureValue(GetExposureTextureHandle(hdParentCamera.currentExposureTextures.previous));
+                                visibleProbe.SetProbeExposureValue(1.0f);
+                                additionalCameraData.deExposureMultiplier = 1.0f / hdParentCamera.GpuDeExposureValue();
```

Comment:
The final goal is for unlit / UI to have an exact luminance when rendered. To achieve this, we 'blow them up' to the opposite exposure, so that when they go through exposure, its the desired nits specified in the unlit/UI material.
With pre exposure this is not necesary, since we are rendering in a consistent relative luminance.

Reply:
Uhm I think for those use cases the objects should be simply rendered after post process, unless I am misunderstanding something here (will come to it later to re-read)

Reply:
Unlit renders directly into the pre post process color buffer. This fixes the issues when having unlit rendering into a planar reflection probe camera. A planar reflection probe camera stores radiance, hence there are no post effects in a PRP camera.

The unlit formula is:
color + emissive*preexposure, which gets rendered into the color buffer of the planar reflction probe.
If the color buffer is pre-exposed, it means that the value that you send in "color" will be exactly that value when sampling the PRP.

If the color buffer of the PRP is not pre-exposed, it means that color will be going through exposure, which will make it darker.
For unlit, i believe the intention is to have that color value be exactly that value, after we apply tone mapping in the main frame.
Thus, if we pre multiply the color times a de exposure value, once the reflection plane gets exposed by the main camera, we will have a sort of accurate unlit color. The alternative is to apply an inverse tonemapping, or even worse, do render planes on the reflection plane itself. 
I think this solution works pretty well for the use case. The only artifact is a slight delay on unlit surfaces in the reflection catching up wtih the current frame's exposure.
Hope that makes sense